python/lsst/meas/mosaic/mosaicTask.py

```diff
@@ -672,7 +672,40 @@ def writeFcr(self, dataRefList):
             try:
                 dataRef.put(exp, "fcr")
             except Exception as e:
-                print "failed to write something: %s" % (e)
+                print "failed to write fcr: %s" % (e)
```

Comment:
Given that the point of this work is to enable us to retire meas_mosaic (by demonstrating that Jointcal is an adequate replacement), I'd rather not spend time on this.